MAUI/Parallax-View/migration.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: Migration from Xamarin to .NET MAUI Parallax View control | Syncfusion
+description: This section explains the features available in the .NET MAUI Parallax View control over Xamarin Parallax View.
+platform: MAUI
+control: SfParallaxView
+documentation: ug
+---
+
+# Migrate from Xamarin.Forms SfParallaxView to .NET MAUI SfParallaxView
+
+The [SfParallaxView]() was created from the scratch using the upgraded APIs and performance of the .NET MAUI graphics library and framework layouts. However, to maintain the consistency of the API naming in MAUI [SfParallaxView](), we renamed some of the APIs. The APIs changed in MAUI [SfParallaxView]() from Xamarin [SfParallaxView]() are detailed as follows:
```

Comment:
The [SfParallaxView]() is created from scratch using the upgraded APIs and performance of the .NET MAUI graphics library and framework layouts. However, to maintain the consistency of the API naming in MAUI [SfParallaxView](), we renamed some of the APIs. The APIs changed in MAUI [SfParallaxView]() from Xamarin [SfParallaxView]() are detailed as follows: